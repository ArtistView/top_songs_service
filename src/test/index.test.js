/**
 * @jest-environment jsdom
 */

import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Enzyme, { shallow, render, mount } from 'enzyme';
import TopFive from '../client/index.jsx';
import {fetch} from 'whatwg-fetch';

describe('TopFive', () => {
  const wrapper = shallow(<TopFive />);

  it('Index renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  })
  it('Has an array in the state', () => {
    expect(Array.isArray(wrapper.state('songs'))).toBe(true)
  })
})

