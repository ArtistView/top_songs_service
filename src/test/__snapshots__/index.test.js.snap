// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopFive Index renders correctly 1`] = `ShallowWrapper {}`;
